LibSWBF2/Chunks/LVL/common/DATA.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "stdafx.h"
+#include "Chunks/GenericChunk.h"
+#include "Chunks/RawData.h"
+#include "Types/Enums.h"
+
+
+namespace LibSWBF2::Chunks::LVL::common
+{
+
+struct LIBSWBF2_API DATA : public GenericChunk<"DATA"_m> {};
```

Comment:
This may contain a static factory method for returning a different ```GenericChunk<"DATA"_m>``` subclass depending on the 4 byte ID of the chunk.